exo-go/features/template/add_template/add_template.feature

```diff
@@ -13,9 +13,15 @@ Feature: adding remote templates
 
 
   Scenario: adding a new service template
-    When running "exo template add boilr-spark https://github.com/tmrts/boilr-spark.git" in my application directory
-    Then my application contains the directory ".exosphere/boilr-spark"
-    And my git repository has a submodule ".exosphere/boilr-spark" with remote "https://github.com/tmrts/boilr-spark.git"
+    When running "exo template add boilr-license https://github.com/tmrts/boilr-license.git" in my application directory
+    Then my application contains the directory ".exosphere/boilr-license"
+    And my git repository has a submodule ".exosphere/boilr-license" with remote "https://github.com/tmrts/boilr-license.git"
+    And my git repository has a submodule ".exosphere/boilr-license" at commit "afb2fa6"
```

Comment:
Hmm okay. I'll just remove this step. I wanted to show it being different from the next test case but I don't think its worth validating its the latest commit.